src/Libraries/RevitNodes/Elements/Group.cs

```diff
@@ -0,0 +1,191 @@
+        /// </summary>
+        /// <param name="location">The point to place the group.</param>
+        /// <param name="groupType">The type of group to place.</param>
+        /// <returns>The new group instance.</returns>
+        public static Group PlaceInstance(Autodesk.DesignScript.Geometry.Point location, Element groupType)
+        {
+            var internalType = groupType.InternalElement as Autodesk.Revit.DB.GroupType;
+            if (internalType == null)
+                throw new InvalidOperationException(String.Format(Properties.Resources.InvalidGroupType, nameof(groupType)));
+
+            XYZ newLocation = location.ToRevitType(true);
+            TransactionManager.Instance.EnsureInTransaction(Document);
+            Group newGroup = Document.Create.PlaceGroup(newLocation, internalType).ToDSType(true) as Group;
+            TransactionManager.Instance.TransactionTaskDone();
+
+            return newGroup;
+        }
+
+        #endregion
+
+        #region public properties
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public Autodesk.DesignScript.Geometry.Point Location
+        {
+            get 
+            {
+                if (this.InternalGroup.IsAttached)
+                    throw new InvalidOperationException(Properties.Resources.AttachedGroupLocation);
+
+                var locationPt = this.InternalGroup.Location as LocationPoint;
+                return locationPt.Point.ToPoint();
+            }
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public Element GroupType
+        {
+            get { return this.InternalGroup.GroupType.ToDSType(true); }
+        }
+
+        /// <summary>
+        /// Indicates wether or not this group is attached to a parent group.
+        /// </summary>
+        public bool IsAttached
+        {
+            get { return this.InternalGroup.IsAttached; }
+        }
+
+        /// <summary>
+        /// Returns the attached detail groups available for this group.
+        /// </summary>
+        public List<Element> AttachedDetailGroup
+        {
+            get 
+            {
+                var attachedGroupId = this.InternalGroup.GetAvailableAttachedDetailGroupTypeIds();
+                return attachedGroupId.Select(id => Document.GetElement(id).ToDSType(true)).ToList();
+            }
+        }
+
+        #endregion
+
+        #region public methods
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        public List<Element> GetMembers()
+        {
+            TransactionManager.Instance.EnsureInTransaction(Document);
+            List<ElementId> memberIds = this.InternalGroup.GetMemberIds().ToList();
+            TransactionManager.Instance.TransactionTaskDone();
+            return memberIds.Select(id => Document.GetElement(id).ToDSType(true)).ToList();         
```

Comment:
same here

Reply:
`this.InternalGroup.GetMemberIds().ToList();` will throw before reaching the return, dont think this will ever be empty though, you can have a group with no members